README.rst

```diff
@@ -88,9 +80,8 @@ in the source folder. Tests restricted to a specific provider can be run by exec
     `new IBMProvider <https://qiskit.org/ecosystem/ibm-provider/stubs/qiskit_ibm_provider.IBMProvider.html>`_
 
     If this is the case, running ``make test`` also executes tests on the ``ibmq`` device.
-    By default tests on the ``ibmq`` device run with ``ibmq_qasm_simulator`` backend
-    and those done by the ``basicaer`` and ``aer`` device are run with the ``qasm_simulator``
-    backend. At the time of writing this means that the test are "free".
+    By default, tests on the ``ibmq`` device run with ``ibmq_qasm_simulator`` backend. At
+    the time of writing this means that the test are "free".
```

Comment:
They are running in the plugin matrix. And yes, we've started seeing warnings about that when running the tests. 

Probably we will want to change our default backend for devices that currently use the `ibmq_qasm_simulator` to be the `aer_simulator` instead, since that's one of their recommended options in their migration documentation. 

For our test, I think almost everything using the `ibmq_qasm_simulator` can also just be replaced with the `aer_simulator` for now. The fake providers and backends sound nice long-term, but just using `aer_simulator` should be pretty quick fix to keep our tests working in the meantime.